modules/ROOT/pages/_partials/commons/common-release-notes.adoc

```diff
@@ -41,7 +42,35 @@ endif::[]
 // BEGIN: Show Maintenance Releases
 
 
-// BEGIN -- Release 2.8.5 (Android-only) -- Mar 2021
+// BEGIN -- Release 2.8.6 (C# only) -- Aug 2021
+ifdef::param-2-8-6-net[]
+
+[#maint-latest]
+[#maint-2.8.6-net]
+== 2.8.6 -- August 2021
+Version 2.8.6 of Couchbase Lite for {param-title} delivers fixes that improve security and performance; specifically resolving memory leak issues.
+
+// === Enhancements
+
+// include::{module-partials}pn-issues-list.adoc[tag=enhancements-2-8-6-net]
+
+
+[#issues-and-resolutions-2-8-6-net]
+=== Issues and Resolutions
+==== Fixed Issues
+include::{module-partials}pn-issues-list.adoc[tag=fixed-2-8-6-net]
+
+==== Known Issues
+
+include::{module-partials}pn-issues-list.adoc[tag=knownissues-2-8-6-net]
+
+
+endif::[]
+// END -- Release 2.8.7 (C#-only) -- Aug 2021
```

Comment:
Shouldn't it be `Release 2.8.6 (C#-only)` ?

Reply:
Will change comment tp 2.8.6